hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -1790,7 +1799,7 @@ int QuectelNcpClient::processEventsImpl() {
     SCOPE_GUARD({ regCheckTime_ = millis(); });
 
     // Check GPRS, LET, NB-IOT network registration status
-    if (ncpId() != PLATFORM_NCP_QUECTEL_BG95_M1) {
+    if (!isQuecLPWADevice()) {
```

Comment:
We need to be careful here and make sure that this works correctly.  I believe the intention of != M1, was to say these commands error on M1 specifically.